index.js

```diff
@@ -7,151 +7,173 @@ import { exit } from 'process'
-            case "$meGithub":
-                await conn.sendMessage(m.key.remoteJid, "https://github.com/rathmerdominik/ITB1-21WaBot", MessageType.text)
-                break;
-            case "$ping":
-                await conn.sendMessage(m.key.remoteJid, "pong", MessageType.text)
-                break;
-        }
+conn.on('chat-update', async (chat) => {
+  if (!chat.hasNewMessage) return
+
+  const m = chat.messages.all()[0]
+
+  if (!m.message || m.key.fromMe) return
+
+  const conv = m.message.conversation
+  const chatNumber = m.key.remoteJid
+
+  if (chat.messages && chat.count) {
+    switch (conv) {
+      case '$suicide': 
+        if (m.participant !== realAdmin) return
+
+        await conn.sendMessage(
+          m.key.remoteJid,
+          'Killing myself, Goodbye!',
+          MessageType.text
+        )
+        conn.close()
+        break
+
+      case '$meGithub':
+        await conn.sendMessage(
+          m.key.remoteJid,
+          'https://github.com/rathmerdominik/ITB1-21WaBot',
+          MessageType.text
+        )
+        break
+
+      case '$ping':
+        await conn.sendMessage(m.key.remoteJid, 'pong', MessageType.text)
+        break
     }
+  }
 })
 
 //be the gigachad
-conn.on('group-participants-update', async group => {
-    console.log(group.jid)    
-    const myGroup = group.jid
-    if(group.action === "add" && myGroup !== groupToSend)
-            await conn.sendMessage(
-                group.jid, 
-                fs.readFileSync("./media/gigachad.jpg"),
-                MessageType.image,
-                    {
-                        mimetype: Mimetype.jpeg, 
-                        caption: "> gets added to group\n> tells everyone to fuck off \n> refuses to elaborate \n> leaves"
-                    }
-                )
-            await conn.groupLeave(group.jid)
-        
-
+conn.on('group-participants-update', async (group) => {
+  console.log(group.jid)
+  const myGroup = group.jid
+  if (group.action === 'add' && myGroup !== groupToSend)
+    await conn.sendMessage(
+      group.jid,
+      fs.readFileSync('./media/gigachad.jpg'),
+      MessageType.image,
+      {
+        mimetype: Mimetype.jpeg,
+        caption:
+          '> gets added to group\n> tells everyone to fuck off \n> refuses to elaborate \n> leaves',
+      }
+    )
+  await conn.groupLeave(group.jid)
 })
 
-conn.on('close', ({reason, isReconnecting}) => {
-    if(reason === "intentional"){
-        console.log("Successfully killed myself! Goodbye")
-        exit(0)
-    } else console.log("Oh shit. I died cuz of "+ reason +"! Do i reconnect? " + isReconnecting)
+conn.on('close', ({ reason, isReconnecting }) => {
+  if (reason === 'intentional') {
+    console.log('Successfully killed myself! Goodbye')
+    exit(0)
+  } else
+    console.log(
+      'Oh shit. I died cuz of ' + reason + '! Do i reconnect? ' + isReconnecting
+    )
 })
-await conn.connect() 
+
+await conn.connect()
+
+// What is this ?
```

Comment:
this is to set the current WAWebVersion. Otherwise the program wont even start as it gets refused from Whatsapp. (Also belongs to these kinds of comments => Does not belong in a pull request diff)